skfda/exploratory/stats/_functional_transformers.py

```diff
@@ -127,7 +67,7 @@ def local_averages(
         data.integrate(interval=((intervals[i], intervals[i + 1]))) / step
         for i in range(n_intervals)
     ]
-    return np.asarray(integrated_data)
+    return np.asarray(integrated_data)[:, :, 0]
```

Comment:
I am actually fine with this function returning a ND-array. It is the transformer, the one which has to return a 2D array. And I think that has to be done using `reshape` and not getting only the 0-th element, as that won't work for more general functions.